Makefile

```diff
@@ -157,32 +157,27 @@ all: binaries
 android-dist: genconfig android
 
 # This is to be called within the docker image.
-docker-genassets: require-npm
+genassets: require-npm
```

Comment:
I can take a look into that Windows build problem, it could be something wrong with the Dockerfile, could you open an issue with the error message and assign it to me?